tests/pyccel/test_pyccel.py

```diff
@@ -294,7 +294,7 @@ def compare_pyth_fort_output( p_output, f_output, dtype=float, language=None):
 #------------------------------------------------------------------------------
 def pyccel_test(test_file, dependencies = None, compile_with_pyccel = True,
         cwd = None, pyccel_commands = "", output_dtype = float,
-        language = None, output_dir = None):
+        language = None, output_dir = None, gpu_available = False):
```

Comment:
I think this parameter should have a different name and default (e.g. `execute_code`).  As written I guess you don't execute any of the C or Fortran tests?

Reply:
Did you work it out yet?

Reply:
Not yet, because I didn't understand what you're trying to say ,sorry

Reply:
can u check now please ff8e26d

Reply:
Much better. This should run the tests correctly now :+1: